### ISSUE TITLE
Accessibility information: links in comments

### ISSUE THREAD

**Issue by gautierchomel** [edrlab/readium-desktop#3518]
All comments at https://github.com/edrlab/thorium-reader/blob/198b8fb33fd606e11d741dd50ec3a6a386517df1/src/renderer/common/components/dialog/publicationInfos/PublicationInfoA11y2.tsx#L94
should be updated to https://www.w3.org/community/reports/publishingcg/CG-FINAL-epub-techniques-20251222/#visual-adjustments

**Comment by panaC** [comment]
I do  not understand your issue, can you be more precise please

**Comment by gautierchomel** [comment]
The link in comments is no longer valid, and we updated the code to stick to the last available version of the recommandation. Then the link in comments should be updated to [https://www.w3.org/community/reports/publishingcg/CG-FINAL-epub-techniques-20251222](https://www.w3.org/community/reports/publishingcg/CG-FINAL-epub-techniques-20251222/). This link should remain available even in the case of updated recommandation and allows us to remember to which version we complky. 

Sorry, the initial comment was too quick and I failed to reference the correct line :)

https://github.com/edrlab/thorium-reader/blob/198b8fb33fd606e11d741dd50ec3a6a386517df1/src/renderer/common/components/dialog/publicationInfos/PublicationInfoA11y2.tsx#L89

**Comment by panaC** [comment]
So in that case this [commit](https://github.com/edrlab/thorium-reader/commit/b5b1d1dd8c6b59e3f3861857b25cf0c5328d981d) fixes the issue.
